redblueapp/buildAndRun.sh

```diff
@@ -1,2 +1,4 @@
 docker build -t securecodingdojo/redblueapp .
-docker run -p 8888:8888 -p 8080:80 -e MASTER_SALT="$CHALLENGE_MASTER_SALT" securecodingdojo/redblueapp
\ No newline at end of file
+CHALLENGE_MASTER_SALT=$(openssl rand -hex 16)
```

Comment:
CHALLENGE_MASTER_SALT must match the training portal. We should be passing it as an environment variable.

Reply:
thanks @paul-ion for your insights. Hence, updated the buildAndRun.sh file